README.md

```diff
@@ -1,2 +1,34 @@
 # docker-spark-gpg
-Docker Alpine image with Spark, Python, Java and gpg. Used for local dev environment.
+Docker Alpine image with Spark, Python, Java and gpg. Used for loacal dev environment.
+
+Instructions
+
+
+1. Run the command in order to mount the export-data folder , the TestData folder and the gpg folder on to the root of the Docker container
+
+'docker run -v /Users/yasserkhan/Desktop/docker-spark-gpg/export-data:/root/export-data -v /Users/yasserkhan/Desktop/docker-spark-gpg/Spark:/root/Spark -v /Users/yasserkhan/Desktop/docker-spark-gpg/gpg:/root/gpg -it docker-spark-gpg :latest bash'
+
+
+NOTE: The gpg keys in the mounted directory will have already been imported. Use the command 'gpg --list-keys' to see a list of all keys imported. There will be a total of 4
+
+
+2. You should now be inside the docker container and be able to see the export-data directory
+
+3. To call this for any single collection we would use:
```

Comment:
Can you remove this line and everything that follows it? This is a specific use case, so is more suited to the Confluence page than the README